addons/mail/models/res_config_settings.py

```diff
@@ -43,3 +46,37 @@ def _compute_fail_counter(self):
             ('date', '>=', previous_date),
             ('state', '=', 'exception'),
         ])
+
+    def open_mail_notification_paynow(self):
```

Comment:
if this is considered a "generic mail layout", can't we use a proper name for that method, regardless of the legacy XML ID it refers to (which is subject to change anyway)?